cluster/s3.yaml

```diff
@@ -1,78 +0,0 @@
-#
```

Comment:
Used in https://github.com/scp756-221/term-project-raccoon-fury/blob/5441bc43792525f5a56bee2280179d4901320816/k8s-tpl.mak#L303

Cannot remove this file?

Reply:
Same as above - we are keeping the templates `s3-tpl.yaml` and `s3-nohealth-tpl.yaml`